docs/ref/models/fields.txt

```diff
@@ -86,14 +86,26 @@ If a field has ``blank=False``, the field will be required.
 
 .. attribute:: Field.choices
 
-A :term:`sequence` consisting itself of iterables of exactly two items (e.g.
-``[(A, B), (A, B) ...]``) to use as choices for this field. If choices are
-given, they're enforced by :ref:`model validation <validating-objects>` and the
-default form widget will be a select box with these choices instead of the
-standard text field.
-
-The first element in each tuple is the actual value to be set on the model,
-and the second element is the human-readable name. For example::
+A dictionary or iterable in the format described below to use as choices for
```

Comment:
I believe this would also need some sort of unification with the former, as in if we chose to say "mapping" to not condition the user to using a dict, we should stick with that terminology across the doc changes :-)

Reply:
Yeah, have tried to use "mapping" more consistently.